### ISSUE TITLE
Update wont install

### ISSUE THREAD

**Issue by Zahkrosis** [Monika-After-Story/MonikaModDev#7926]
Simply put, I tried 5 times, downloading, extracting, putting in the files and folders, the update still wont go through. Still tells me "An update is available"

**Comment by toonziz** [comment]
send a screen shot of the settings menu, the game files, and the update screen.

**Comment by ThePotatoGuy** [comment]
check this [FAQ entry here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#mas-keeps-telling-me-that-theres-an-update-but-when-i-click-update-now-it-says-mas-is-up-to-date)

**Comment by Zahkrosis** [comment]
[image: 20210926164731_1.jpg]
[image: 20210926165108_1.jpg]
[image: Screenshot (179).png]

**Comment by ThePotatoGuy** [comment]
can't see those images, did it work for you?

**Comment by Zahkrosis** [comment]
Yes, it worked. I can see them.

Point is, no matter when or how I move the files in, the in-game ALWAYS
says "Update available" once launched.

**Comment by ThePotatoGuy** [comment]
No, my "did it work for you" question was about if the FAQ entry worked. I still cannot see your images. Since you're replying by email, maybe try adding as attachments? or pasting directly in the message?
